models/stg_linkedin_ads__creative_history.sql

```diff
@@ -28,11 +28,11 @@ with base as (
         source_relation,
         id as creative_id,
         campaign_id,
-        coalesce(intended_status, status) as status,
+        coalesce(status, intended_status) as status,
```

Comment:
I understand the changes to the `last_modified_at` fields and window function. However, we never fully checked the `status` and `created_at` fields and if this update was required. 

We do have these fields in our internal schema. Can you add to your validation workbook confirmation that the results of the output between the dev branch and prod do not unexpectedly change following this update.

Reply:
I added to my validation that the status and created_at fields were not affected, however I only switched them so the pattern of the coalesces would be consistent in this model. I am open to switching them back to the original way if you think that would be better!